benchmark/bench_CQRRPT/CQRRPT_speed_comparisons.cc

```diff
@@ -207,7 +207,7 @@ int main() {
                                       + ".dat", std::fstream::app);
 
     for (;n_start <= n_stop; n_start *= 2) {
-        res = call_all_algs<double, r123::Philox4x32>(m_info, numruns, n_start, all_data, state_constant);
+        res = call_all_algs<double>(m_info, numruns, n_start, all_data, state_constant);
```

Comment:
``double`` can be inferred from ``all_data``. 

It seems that both template parameters to ``call_all_algs`` can be inferred from argument values. So I suggest CTRL+F for "call_all_algs<" to find all places where you're (unnecessarily) specifying template parameters.

Reply:
Fixed.